data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,272 @@
+				accept
+	
+	on visit
+		dialog `You look for the young Hai, but realize that he took a ride on one of your escorts! Better depart and wait for your escorts to arrive in this star system.`
+	on complete
+		payment 100000
+		dialog
+			`You need not have worried about the reception the Unfettered youth would receive here on Hai-home. Scarcely minutes after you hesitantly contact the Hai government, a happy crowd has gathered around your ship. When he steps out of the hatchway, he looks stunned at being welcomed so warmly. One of the Hai governors thanks you for transporting him, and pays you <payment>.`
+		"reputation: Hai" += 10
+		"reputation: Hai (Wormhole Access)" += 10
+		"reputation: Hai Merchant" += 10
+		"reputation: Hai Merchant (Human)" += 10
+		"reputation: Hai Merchant (Sympathizers)" += 10
+
+
+
+ship "Ancient Ladybug"
+	swizzle 0
+	sprite "ship/hai ladybug"
+	thumbnail "thumbnail/hai ladybug"
+	attributes
+		category "Medium Warship"
+		"cost" 8950000
+		"shields" 12000
+		"hull" 3300
+		"required crew" 11
+		"bunks" 33
+		"mass" 300
+		"drag" 5.5
+		"heat dissipation" .85
+		"fuel capacity" 900
+		"cargo space" 91
+		"outfit space" 460
+		"weapon capacity" 170
+		"engine capacity" 120
+
+		# Expensive features mentioned in description, nerfed:
+		"ramscoop" 0.2
+		"cargo scan power" 4
+		"cargo scan efficiency" 14
+		"outfit scan power" 15
+		"outfit scan efficiency" 12
+		"tactical scan power" 25
+		"atmosphere scan" 80
+		"asteroid scan power" 25
+		
+		weapon
+			"blast radius" 100
+			"shield damage" 1000
+			"hull damage" 500
+			"hit force" 1500
+	outfits
+		`"Biroo" Atomic Steering`
+		`"Biroo" Atomic Thruster`
+		"Boulder Reactor"
+		"Bullfrog Anti-Missile" 2
+		"Cargo Expansion" 2
+		"Hai Diamond Regenerator"
+		"Hai Fissure Batteries"
+		"Hai Williwaw Cooling"
+		Hyperdrive
+		"Ion Cannon" 3
+		"Quantum Keystone"
+
+	gun 0 -69.5 "Ion Cannon"
+	gun -12 -68.5 "Ion Cannon"
+	gun 12 -68.5 "Ion Cannon"
+	turret -36 3 "Bullfrog Anti-Missile"
+	turret 36 3 "Bullfrog Anti-Missile"
+	engine -36 62 0.8
+	engine 36 62 0.8
+	explode "tiny explosion" 20
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 10
+	"final explode" "final explosion large"
+	description "This sensor-laden exploration vessel hails back to a more peaceful era of the Hai. Though they have no true ramscoop, a technology the Hai never developed, their built-in collection gills magnify the natural ramscoop present in all ships. These expensive features are the reason the ship has been replaced with the Lightning Bug."
+	description "However, this ship is very useful as an agile cargo hauler, able to defend itself in small skirmishes and bring cargo to long distances. However you suspect that it does not operate at its full potential, as if it stayed in the ocean too long... Still, this ship is clearly proof of the durability of Hai technology."
+
+mission "Old but not that old"
```

Comment:
This should probably be the `name`, and define the mission differently.

Reply:
What for?
I'm still not sure on the way to define all Unfettered missions honestly but since this is a side one it doesnt really matter (also its mostly a ref ^^)

Reply:
I think Hai Legacy sounds a little better.